app/src/main/java/org/mozilla/focus/autocomplete/AutocompletePreference.kt

```diff
@@ -1,36 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.focus.autocomplete
-
-import android.content.Context
-import android.util.AttributeSet
-import android.widget.TextView
-import androidx.preference.Preference
-import androidx.preference.PreferenceViewHolder
-import org.mozilla.focus.R
-import org.mozilla.focus.ext.settings
-
-/**
- * Autocomplete preference that will show a sub screen to configure the autocomplete behavior.
- */
-class AutocompletePreference(context: Context?, attrs: AttributeSet?) : Preference(context, attrs) {
```

Comment:
This seems to have been unused for some time.
Nice to have cleaned it though it would've been nice to be in a separate commit.
If there are more such situations in code maybe a ticket just for cleaning unused classes and resources would be useful.